src/lib.rs

```diff
@@ -140,17 +140,53 @@ pub fn installed_versions() -> Result<Vec<Version>, SolcVmError> {
     Ok(versions)
 }
 
+/// Blocking version of [`all_versions`]
+#[cfg(feature = "blocking")]
+pub fn block_all_versions() -> Result<Vec<Version>, SolcVmError> {
```

Comment:
Tiny suggestion to be consistent with the `blocking_*` naming you followed :)

```suggestion
pub fn blocking_all_versions() -> Result<Vec<Version>, SolcVmError> {
```